### DISCUSSION TITLE
Overriding rules on a particular row

### DISCUSSION THREAD

**Discussion by dahosek** [lvjr/tabularray#233] [General]
Is there any way besides using, e.g., `vline{1} = {2}{width=1pt}` to override a rule on a particular row? As awkward as the `tabular` syntax of using `\multicolumn` for that purpose, it has the advantage of putting the formatting near the place where the formatting is overridden. I've had little success experimenting with `\vline`, `\SetVline` and `\SetVlines` to this end. I'm also wondering what the function of `\rline` is meant to be?

**Comment by lvjr** [comment]
The syntax of `\rline` is similar to that of  `\cline`, but you can not set last vline, since there is no place for it.
```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
\begin{tblr}{lcr}
\rline{1} Alpha & \rline{1-2} Beta & \rline{2-3} Gamma  \\
        Epsilon &             Zeta &             Eta    \\
           Iota &            Kappa &             Lambda \\
\end{tblr}
\end{document}
```
![image](https://user-images.githubusercontent.com/4239949/163545277-76ddf92d-0486-42e9-bbe1-06945c8d0844.png)

**Comment by lvjr** [comment]
It is in my todo list to allow setting borders with `\SetCell`, `\SetRow`, and `\SetColumn` commands (in their optional arguments).
